packages/esbuild-plugins/src/remove-strict.ts

```diff
@@ -8,6 +8,7 @@ export function removeStrict(): Plugin {
             build.initialOptions.write = false;
             build.onEnd((result) => {
                 result?.outputFiles?.forEach(file => {
+                    console.log(file.path);
```

Comment:
console.log left here? or is it supposed to be, and should it have some prefix about what's happening?